### PR TITLE
Added IRCCloud 0.1.7

### PR DESCRIPTION
**Pull request by cjp**
#### Adding a new cask
- [X] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [X] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [X] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [X] Commit message includes cask’s name.
- [X] `brew cask audit --download irccloud` is error-free.
- [X] `brew cask style --fix irccloud` left no offenses.
- [X] `brew cask install irccloud` worked successfully.
- [X] `brew cask uninstall irccloud` worked successfully.

### CODE REVIEW

File: Casks/irccloud.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'irccloud' do
+  version '0.1.7'
+  sha256 'f013072b0aff656f11812eb5d2d7f7bc9408177f981798a740c4f1b028f3040e'
+
+  url 'https://github.com/irccloud/irccloud-desktop/releases/download/v0.1.7/IRCCloud-0.1.7.dmg'
```
Comment (fanquake) [inline Casks/irccloud.rb]: You should be using #{version} in the url stanza

Comment (vitorgalvao) [inline Casks/irccloud.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

Comment (vitorgalvao) [inline Casks/irccloud.rb]: What @fanquake means is in the end it should look like `url "https://github.com/irccloud/irccloud-desktop/releases/download/v#{version}/IRCCloud-#{version}.dmg"`.

Comment (vitorgalvao) [inline Casks/irccloud.rb]: I don’t see any indication on the site this is official.

**Comment by vitorgalvao** [comment]
Closing for lack of reply. You may submit again in the future, with the requested fixes and answer to the question.

**Pull request closed without merging**